torch/_subclasses/fake_tensor.py

```diff
@@ -152,10 +152,10 @@ def unset_fake_temporarily() -> Generator[Optional[TorchDispatchMode], None, Non
 
 
 def get_plain_tensors(
-    subclass: Tensor, out_append_list: Optional[List[Tensor]] = None
-) -> List[Tensor]:
+    subclass: Tensor, out_append_list: Optional[List[Union[Tensor, int, SymInt]]] = None
+) -> List[Union[Tensor, int, SymInt]]:
```

Comment:
hmm, the type signature here is a bit confusing, since we never actually append ints/SymInts to the list in this function. I guess you needed this because in the `out_append_list=` case, the  list we pass in might have symints in it already?

Instead, what do you think of: just refactoring this function to **always** accept an output list to append to, and mandating that anybody using this API must pass in their own list (from a quick grep there are only 2 call sites of this function, both within AOTAutograd)